pep-0001.txt

```diff
@@ -242,37 +245,86 @@ on GitHub.
 
 As updates are necessary, the PEP author can check in new versions if they
 (or a collaborating developer) have write access to the `PEP repository`_.
-
-After a PEP number has been assigned, a draft PEP may be discussed further on
-python-ideas (getting a PEP number assigned early can be useful for ease of
+Getting a PEP number assigned early can be useful for ease of
 reference, especially when multiple draft PEPs are being considered at the
-same time). Eventually, all Standards Track PEPs must  be sent to the
-`python-dev list <mailto:python-dev@python.org>`__ for review as described
-in the next section.
+same time.
 
 Standards Track PEPs consist of two parts, a design document and a
 reference implementation.  It is generally recommended that at least a
 prototype implementation be co-developed with the PEP, as ideas that sound
 good in principle sometimes turn out to be impractical when subjected to the
 test of implementation.
 
+
+Discussing a PEP
+----------------
+
+As soon as a PEP number has been assigned
+and the draft PEP is committed to the `PEP repository`_,
+a discussion thread for the PEP should be created
+to provide a central place to discuss and review its contents, and the
+PEP should be updated so that the ``Discussions-To`` header links to it.
+
+The PEP authors (or sponsor, if applicable) may select any reasonable venue
+for the discussion, so long as the the following criteria are met:
+
+* The forum is appropriate to the PEP's topic.
+* The thread is publicly available on the web so that all interested parties
+  can participate.
+* The discussion is subject to the `Python Community Code of Conduct
+  <https://www.python.org/psf/conduct/>`_.
+* A direct link to the current discussion thread is provided in the PEP
+  under the ``Discussions-To`` header.
+
+Typically, the `Python-Dev`_ mailing list and the
+`PEPs category`_ of the `Python Discourse`_ are good choices for most PEPs,
+while some specialized topics have specific venues, such as
+`Typing-SIG`_ for typing PEPs or the `Packaging category`_ on the Python
+Discourse for packaging PEPs. If the PEP authors are unsure of the best venue,
+the PEP Sponsor and PEP editors can advise them accordingly.
+
+If the chosen venue is not the `Python-Dev`_ mailing list,
+a brief announcement should be posted there when the draft PEP is
+committed to the PEP repository and available on the PEP website,
+with a link to the rendered PEP and the to canonical Discussions-To thread.
```

Comment:
Nit: `and the to` -> `and to the`

Reply:
Good catch, thanks! I'll fix it in my forthcoming followup PR (see below).